docs/redis-clients/_index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Redis client libraries"
+linkTitle: "Clients"
+description: Connect your application to a Redis database
+weight: 4
+
+---
+
+Here, you will learn how to connect your application to a Redis database. If you're new to Redis, you might first want to understand how to:
+
+* [Get started with Redis](/docs/getting-started/) 
```

Comment:
Throwing all those links to the new Redis User doesn't make any sense.

We need to help new users to install an all-in-one option which is RedisStack with RedisInsight:

> If you're new to Redis, you might first want to [Install Redis with Redis Stack and RedisInsight](/docs/stack/get-started/install)

Also, we need to redirect users looking for GUI Client for Redis to the RedisInsight page.

Reply:
[](url)